feat: adding remaining python functions, removing unnecessary dependencies, standardizing env variables (#54)
appwrite/demos-for-functions: python/welcome-email/README.md

# 📧 Welcome Email 

> Sends the welcome email to the newly registered users to the app according to the template provided.

Reviewer: ```suggestion
A sample Python Cloud Function for sending a welcome email to a newly registered user.
```

## 📝 Environment Variables
Go to Settings tab of your Cloud Function. Add the following environment variables.

> * **MAILGUN_API_KEY** =  API Key for Mailgun
> * **MAILGUN_DOMAIN** = Domain Name from Mailgun

Reviewer: ```suggestion
* **MAILGUN_API_KEY** -  API Key for Mailgun
* **MAILGUN_DOMAIN** - Domain Name from Mailgun
```

## 🚀 Building and Packaging

To package this example as a cloud function, follow these steps.

```bash
$ cd demos-for-functions/python/welcome-email

$ PIP_TARGET=./.appwrite pip install -r ./requirements.txt --upgrade --ignore-installed 
```

* Ensure that your folder structure looks like this 
```
.
├── .appwrite/
├── main.py
└── requirements.txt
```

* Create a tarfile

```bash
$ cd ..
$ tar -zcvf code.tar.gz welcome-email
```

> * Upload the tarfile to your Appwrite Console and use the following entrypoint command
> 
> ```bash
> python main.py
> ```
> 
> * Navigate to the Overview Tab of your Cloud Function > Deploy Tag
> * Input the command that will run your function (in this case `python main.py`) as your entrypoint command
> * Upload your tarfile 
> * Click 'Activate'

Reviewer: ```suggestion
* Navigate to the Overview Tab of your Cloud Function > Deploy Tag
* Input the command that will run your function (in this case `python main.py`) as your entrypoint command
* Upload your tarfile 
* Click 'Activate'
```
